src/compiler/crystal/semantic/literal_expander.cr

```diff
@@ -638,7 +638,7 @@ module Crystal
         end
       end
 
-      Call.new(cond, "===", right_side).at(obj)
+      Call.new(cond, "===", right_side)
```

Comment:
Quick question: why is this no longer needed? (asking just out of curiosity, as I'm not particularly familiar with this part of the compiler)